src/telemetry/telemetry.ts

```diff
@@ -9,11 +9,13 @@ class Telemetry implements ITelemetry {
   private reactPlugin: any;
 
   constructor() {
+    const { mscc } = (window as any);
+
     this.reactPlugin  = new ReactPlugin();
     this.config = {
-      instrumentationKey: process.env.REACT_APP_INSTRUMENTATION_KEY,
+      instrumentationKey: this.getInstrumentationKey(),
       disableExceptionTracking: true,
-      disableTelemetry: this.areWeInDev() ? false : true,
+      disableTelemetry: this.getInstrumentationKey() ? false : true,
```

Comment:
From what I understand, we disable Telemetry when we do not find an instrumentation key. Right?
However it seems that even without a key in the env file, it does not disable telemetry.

I also noticed that if the key is empty, it still does not disable

Reply:
That is interesting, following up on your DM to learn more about why telemetry is not disabled.

Reply:
Resolving this.

We have to stop and start the server for it to pick up changes from the .env file